src/models/searchdelegate.cpp

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
looking at what this does, please remove it altogether, this should be handled by using KTextEditor instead